### PR TITLE
Update configure-endpoints-gp.md

### LINKED ISSUE #12793: Offboard devices using Group Policy Step 6 Documentation is not correct
Step 6 in Offboard devices using Group Policy is not correct. You can't use BUILTIN\SYSTEM and select "Run whether user is logged on or not and check the Run with highest privileges checkbox. 
This should be modified to **NT AUTHORITY\SYSTEM** and match the Onboarding Step.

 **"In the Task window that opens, go to the General tab. Under Security options click Change User or Group and type SYSTEM and then click Check Names then OK. NT AUTHORITY\SYSTEM appears as the user account the task will run as."**

Offboard devices using Group Policy-
Select Run whether user is logged on or not and check the Run with highest privileges check box.
6. In the Task window that opens, go to the General tab. Choose the local SYSTEM user account **(BUILTIN\SYSTEM)** under Security options.
7. Select Run whether user is logged on or not and check the Run with highest privileges check-box.

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 523651c3-a318-2369-6892-d13e04991596
* Version Independent ID: 624c5e05-f87d-1bd4-bc87-bd55dcac1d86
* Content: [Onboard Windows devices to Microsoft Defender for Endpoint via Group Policy](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/configure-endpoints-gp?view=o365-worldwide)
* Content Source: [microsoft-365/security/defender-endpoint/configure-endpoints-gp.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/configure-endpoints-gp.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @siosulli
* Microsoft Alias: **siosulli**

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to Step 6

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12793

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/configure-endpoints-gp.md [hunk 1]
```diff
@@ -219,7 +219,7 @@ For security reasons, the package used to Offboard devices will expire 30 days a
 
 5. Right-click **Scheduled tasks**, point to **New**, and then click **Immediate task**.
 
-6. In the **Task** window that opens, go to the **General** tab. Choose the local SYSTEM user account (BUILTIN\SYSTEM) under **Security options**.
+6. In the **Task** window that opens, go to the **General** tab. Under **Security options** click Change User or Group and type SYSTEM and then click Check Names then OK. NT AUTHORITY\SYSTEM appears as the user account the task will run as.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-endpoints-gp.md]:
```suggestion
6. In the **Task** window that opens, go to the **General** tab under **Security options** and select **Change User or Group**, enter SYSTEM, then select **Check Names** and then **OK**. NT AUTHORITY\SYSTEM appears as the user account that the task will run as.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@siosulli Please merge this PR. Thanks

**Pull request merged**